ocaml/src/albamgr_protocol.ml

```diff
@@ -219,12 +222,30 @@ module Protocol = struct
         =
         let my_compare x y = compare y x in
         let kind =
-          let ips, port = get_ips_port osd.kind in
-          let ips' = Option.get_some_default ips ips' in
-          let port' = Option.get_some_default port port' in
+          let ips, port, tls = get_conn_info osd.kind in
+          let conn_info' =
+            let ips'  = Option.get_some_default ips ips' in
+            let port' = Option.get_some_default port port' in
+            let real_port, use_tls =
+              let open Tiny_json in
+              match other' with
+              | None -> port', tls
+              | Some other' ->
+                 try
+                   let r = Json.parse other' in
+                   let tlsPort = Json.as_int (Json.getf "tlsPort" r)  in
```

Comment:
I was already wondering why I saw no UpdateOsds2 ... now I know how you did it :)